modules/monitoring-disabling-the-local-alertmanager.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+[id="disabling_the_local_alertmanager_{context}"]
+= Disabling the local Alertmanager
+
+A default installation of the {product-title} monitoring stack includes in the `openshift-monitoring` project an Alertmanager instance that routes alerts from Prometheus instances.
+
+You can disable this Alertmanager instance by configuring the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project.
+You might do so in single-node deployments that do not need to deploy a local Alertmanager cluster because alerts are centralized at the core (for example, in the hub cluster) or because running Alertmanager locally would take resources away from user workloads and add management overhead.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created the `cluster-monitoring-config` `ConfigMap` object.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Edit the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+. Add `enabled: false` for the `alertmanagerMain` component under `data/config.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    alertmanagerMain:
+      enabled: false
+----
+
+. Save the file to apply the changes. The Alertmanager instance is disabled automatically when you apply the change.
++
+[WARNING]
+====
+When changes are saved to the `cluster-monitoring-config` `ConfigMap` object, the pods and other resources in the `openshift-monitoring` project might be redeployed. The running monitoring processes in that project might also be restarted.
```

Comment:
Can you rephrase this using the active voice? E.g., "Saving changes to the `cluster-monitoring-config` `ConfigMap` object might redeploy the pods and other resources in the `openshift-monitoring` project and restart the running monitoring processes in that project."